static/src/stylesheets/module/_adslot.scss

```diff
@@ -69,8 +69,6 @@
     .sticky-ad-banner & {
         z-index: 1022;
         width: 100%;
-        transform: translateZ(0);
-        will-change: transform;
```

Comment:
To have any positive effect, the `will-change` property should be set before the animation, and then removed after it has ended.